### PR TITLE
Aliaksandr_Horosko_Hw2_Task4

### PR DESCRIPTION
**Pull request by Alexhorosko**

### CODE REVIEW

File: homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
+package by.alhr.hw2task4.bean;
+
+import java.util.Objects;
+
+public class RandomGenerator {
+    public RandomGenerator() {
+        return;
```
Comment (javaGuruBY) [inline homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java]: no need for return here

File: homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        this.c = c;
+    }
+
+    public RandomGenerator(double a, double b, double c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+    }
+
+    double a = Math.random() * 3;
```
Comment (javaGuruBY) [inline homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java]: fields should be at the top of the Class

File: homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    double b = Math.random() * 8;
+    double c = Math.random() * 12;
+    public double aDouble (){
```
Comment (javaGuruBY) [inline homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java]: aDouble what? 
get or set?

re-write using javaBean convention

Comment (Alexhorosko) [reply homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java]: correct

File: homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java (hunk 2 above)
Comment (Alexhorosko) [reply homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java]: correct

File: homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java (hunk 1 above)
Comment (Alexhorosko) [reply homework/Aliaksandr_Horosko/Hw2_Task4/src/main/java/by/alhr/hw2task4/bean/RandomGenerator.java]: correct

**Pull request merged**